### ISSUE TITLE
Test accuracy is lower than in the paper on Washington RGB-D

### ISSUE THREAD

**Issue by mhwasil** [MRLoghmani/rcfusion#2]
Hallo,
I run the code on Nvidia K80 using tnesorflow 1.15 and got around 76% test accuracy for trial-1 of Washington RGB-D dataset.
I did not preprocess the depth image and did not change the hyperparameters.

I know that there is a warning already that the results may fluctuate on tensorflow 1.10 or higher, but considering the test accuracy drop I got, is that because of the tf version?

Did you do any preprocessing (like normal estimation, etc) for the depth images of W RGB-D dataset?, you do have colorization before training in the paper, I missed this.

Maybe that's why I got 76%.

Since you use the colorization method, do you also open-source the code?

Thank you

![Screenshot from 2019-12-30 17-18-51](https://user-images.githubusercontent.com/22024670/71590444-cf9bc580-2b28-11ea-9f97-b960e1eb039c.png)

**Comment by MRLoghmani** [comment]
Hi and thank you for your interest in the repository.

Yes, there is a pre-processing phase which is especially important for the depth data. As mentioned in our paper, we apply the same pre-processing that is described in Section III.B of [Aakerberg et al.]: https://vbn.aau.dk/ws/files/263084822/Ensemble_FusionNet_IPTA.pdf

In order to speed up your work, I have put together the files I have used to run the experiments on the first split of the Washington RGB-D dataset. If you are interested in trying RCFusion on your own dataset, I could provide you with some code to do your own pre-processing, but I anyway recommend checking out the paper.

WRGB-D files:
> Pre-processed dataset: https://data.acin.tuwien.ac.at/index.php/s/YKZQmoRtWaAcU91
> Split files, network weights and mean files: https://data.acin.tuwien.ac.at/index.php/s/eRXpoDrwSMxjAWq

Please let me know if there are some issues with the files.

Good luck!

**References:**
_[Aakerberg et al.]    A. Aakerberg, K. Nasrollahi, and T. Heder, “Improving a deep learning based rgb-d object recognition model by ensemble learning,” in IEEE International Conference on Image Processing Theory, Tools and Applications, 2017, pp. 1–6._

**Comment by jhss** [comment]
Hi

I want to apply your pre-processing technique to other dataset.
I read the paper Aakerberg et al, but It's hard for me to implement it.
Would you provide pre-processing code?
